roles/edpm_libvirt/molecule/vagrant/molecule.yml

```diff
@@ -0,0 +1,36 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: vagrant
+  provider:
+    name: libvirt
+  provision: no
+  parallel: true
+  # default_box: 'generic/rocky9'
+  default_box: 'generic/centos9s'
+platforms:
+- name: compute-1
+  memory: 8192
+  cpus: 8
+  provider_options:
+    cpu_mode: 'host-passthrough'
+    nested: true
+    machine_type: 'q35'
+  groups:
+    - compute
+provisioner:
+  name: ansible
+  playbooks:
+    converge: ../default/converge.yml
+    prepare: ../default/prepare.yml
+verifier:
+  name: ansible
+scenario:
```

Comment:
we chatted about this before and at the time i thought i would need to modify the ci-framework before we could skip some scenarios

i think this might work instead so I'm going to just submit this as a pr and see what happens.

Reply:
:+1:

Reply:
so test_sequence: [] does in fact work so ill mark this as resolved